src/components/enketo/preview.vue

```diff
@@ -10,16 +10,16 @@ including this file, may be copied, modified, propagated, or distributed
 except according to the terms contained in the LICENSE file.
 -->
 <template>
-  <a v-if="disabledDescription == null" :class="enketoClass" :href="enketoPath"
+  <a v-if="disabledDescription == null" class="enketo-preview btn" :class="btnClass" :href="previewPath"
```

Comment:
I introduced `enketoClass` recently in #1249, but when I did so, I thought for some reason that this link would always go to Enketo. That's not the case: it links to the canonical preview path (either Enketo or Web Forms depending on the setting). Given that, I think it's a good idea not to use names like `enketoClass` or `enketoPath`. I want to minimize references to Enketo that could equally apply to Web Forms. (On that note, we should probably rename this component to something like `FormPreviewButton`. That's for a different PR though.)